Source/WebCore/editing/markup.cpp

```diff
@@ -223,6 +223,76 @@ String sanitizeMarkup(const String& rawHTML, MSOListQuirks msoListQuirks, std::o
     return sanitizedMarkupForFragmentInDocument(WTFMove(fragment), *stagingDocument, msoListQuirks, rawHTML);
 }
 
+UserSelectNoneStateCache::UserSelectNoneStateCache(TreeType treeType)
+    : m_useComposedTree(treeType == ComposedTree)
+{
+    ASSERT(treeType == Tree || treeType == ComposedTree);
+}
+
+ContainerNode* UserSelectNoneStateCache::parentNode(Node& node)
+{
+    if (UNLIKELY(m_useComposedTree))
```

Comment:
Is this code so hot we need to explicitly use `UNLIKELY`?

Reply:
Actually, we probably don't need LIKELY/UNLIKELY here.